.travis.yml

```diff
@@ -16,3 +16,4 @@ script:
   - go generate -x ./... && git diff --exit-code; code=$?; git checkout -- .; (exit $code) # Check that go generate ./... produces a zero diff; clean up any changes afterwards.
   - go tool vet .
   - go test -v -race ./...
+  - go test -v -tags=integration -run=^$ ./test/integration # Check that integration test builds successfully, but don't run any of the tests (they hit live GitHub API).
```

Comment:
I saw that, but guess I didn't fully understand it.  Also, I didn't know why `go install` was used instead of `go build`. Oh, OK.  I think I get it now. No need to elaborate, thanks.